internal/bgp/frrk8s/frrk8s.go

```diff
@@ -48,10 +47,22 @@ type session struct {
 	logger         log.Logger
 }
 
-// sessionName() defines the format of the key of the 'sessions' map in
-// the 'frrState' struct.
+// sessionName() defines the format of a map key.
 func sessionName(s session) string {
-	baseName := fmt.Sprintf("%d@%s-%d@%s", s.PeerASN, s.PeerAddress, s.MyASN, s.SourceAddress)
+	f := func(ASN uint32, dynamicASN string) string {
```

Comment:
can we just use the contents of this function to set asn? this is a bit confusing, especially since it's used once